website/src/components/Showcase/Showcase.js

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect } from 'react';
+import Card from './Card';
+import styles from '../../pages/showcase.module.css';
```

Comment:
Use individual module css. Why do you use shared css file for all components?

Reply:
added your fixes to this - thanks - made sure to split up in homepage edits.